dataactvalidator/validation_handlers/validator.py

```diff
@@ -782,3 +789,27 @@ def validateTAS(fieldsToCheck, tasFields, record, interfaces, fileType):
         else:
             # Multiple instances of same TAS, something is going wrong
             raise ResponseException("TAS check is malfunctioning",StatusCode.INTERNAL_ERROR)
+
+    @staticmethod
+    def validateFileBySql(submissionId,fileType,interfaces):
+        # Pull all SQL rules for this file type
+        fileId = interfaces.validationDb.getFileId(fileType)
+        rules = interfaces.validationDb.session.query(RuleSql).filter(RuleSql.file_id == fileId).all()
```

Comment:
We probably also want to filter for cross_file_flag = False here. In my initial stab at the sql validation table, I added fileType in the cross file rules A18 and A19 to indicate the target file being validated, and we don't want those rules to be returned in this query.